grammar/operator.md

```diff
@@ -584,7 +584,7 @@ x // 1
 这种跳过第二个运算子的机制，被称为“短路”。有些程序员喜欢用它取代`if`结构，比如下面是一段`if`结构的代码，就可以用且运算符改写。
 
 ```javascript
-if (i !== 0 ) {
+if (i != 0 ) {
```

Comment:
确实，我改一下。

2016-07-18 14:28 GMT+08:00 troy351 <notifications@github.com>:

> In grammar/operator.md
> <https://github.com/ruanyf/jstutorial/pull/149#discussion_r71103718>:
>
> > @@ -584,7 +584,7 @@ x // 1
> >  这种跳过第二个运算子的机制，被称为“短路”。有些程序员喜欢用它取代`if`结构，比如下面是一段`if`结构的代码，就可以用且运算符改写。
> >
> >  ```javascript
> > -if (i !== 0 ) {
> > +if (i != 0 ) {
>
> 反了吧，应该是if (i) {
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/ruanyf/jstutorial/pull/149/files/db1b7f030a943bf3dd29cb1420db29e481002244#r71103718>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA3Q2jtZCJEu21qTKW6i3k_vY9JLStpwks5qWx0WgaJpZM4JOPZ_>
> .
>